client/src/components/Download.js

```diff
@@ -45,10 +51,41 @@ export const Download = ({ icon, resource }) => {
     setPublicComputedFile(file)
   }
 
+  const handleToggleFile = () => {
+    setResource(project)
+    setPublicComputedFile(project.computed_files[0])
+    setProject(null)
+    setToggleFile(true)
+    setInital(false)
+    setDownload(false)
+  }
+
+  useEffect(() => {
+    if (initial || (!initial && !toggleFile)) setResource(initialResource)
+
+    setPublicComputedFile(
+      mutltipleComputedFiles ? null : resource.computed_files[0]
```

Comment:
lets write a helper her to get `getDefaultComputedFile` which returns resource.computed_files[0]